docs/includes/topic_basic-environment-reqs.rst

```diff
@@ -0,0 +1,99 @@
+:orphan: true
+:hidden:
+
+Basic Environment Requirements for F5 LBaaSv2
+=============================================
+
+This document provides the minimum basic requirements for using F5® LBaaSv2 in OpenStack |openstack|.
+
+OpenStack Requirements
+----------------------
+
+The OpenStack installation guides cover the requirements for specific environments.
+
+    - `Installation Guide for Red Hat Enterprise Linux 7 and CentOS 7`_
+    - `Installation Guide for Ubuntu 14.04 (LTS)`_
+    - `OpenStack Installation Guide for openSUSE and SUSE Linux Enterprise`_
+
+We recommend that you install and configure the following OpenStack services. Each of these is necessary for one or more F5 OpenStack integrations.
+
++-------------+-------------------------+-----------------------+
+| Service     | Description             | F5 Integration        |
++=============+=========================+=======================+
+| `Nova`_     | compute service         | LBaaSv2, Heat         |
++-------------+-------------------------+-----------------------+
+| `Neutron`_  | networking              | LBaaSv2               |
++-------------+-------------------------+-----------------------+
+| `Keystone`_ | identity                | LBaaSv2, Heat         |
++-------------+-------------------------+-----------------------+
+| `Glance`_   | image service           | Heat                  |
++-------------+-------------------------+-----------------------+
+| `Horizon`_  | dashboard               | LBaaSv2 [#]_, Heat    |
++-------------+-------------------------+-----------------------+
+| `Heat`_     | orchestration           | Heat                  |
++-------------+-------------------------+-----------------------+
+| `Barbican`_ | key management          | LBaaSv2               |
++-------------+-------------------------+-----------------------+
+
+
+BIG-IP Requirements
+-------------------
+
+.. important::
+
+    You must have the appropriate `license`_ for the BIG-IP features you wish to use.
+
+
+.. table:: BIG-IP Requirements [#]_
+
+    +----------------------------+--------+------------+----------------+-------------+-----------------+
+    | Deployment [#]_            | NICs   | VLANs [#]_ | Tunnels [#]_   | VTEPs [#]_  | License         |
```

Comment:
I think footnote 3 applies only to the NICs column.  Hosts in the same cloud will share N VLANs (carried over the appropriate NIC).